packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -179,9 +179,11 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
         exp: exp,
         enable_recording: scalePlan === "true" && !!hasTeamPlan === true ? "cloud" : undefined,
         enable_transcription_storage: !!hasTeamPlan,
-        permissions: {
-          canAdmin: ["transcription"],
-        },
+        ...(!!hasTeamPlan && {
+          permissions: {
+            canAdmin: ["transcription"],
+          },
+        }),
```

Comment:
Don't know why create meeting failed with an error invalid type of enable_transcription_storage because !!hasTeamPlan would always be a boolean value

Reply:
adding default somehow fixes it  enable_transcription_storage: z.boolean().default(false),

Reply:
strange indeed, I would directly do this 

```ts

 const hasTeamPlan = Boolean(await prisma.membership.findFirst({
      where: {
        userId: event.organizer.id,
        team: {
          slug: {
            not: null,
          },
        },
      },
    }));
```